app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    components = command.split()
+    first_part, second_part, third_part = components
+    if len(components) == 3 and first_part == "mv":
+
+        file_name = second_part
+        path = third_part
+        dirs = third_part.split("/")
+
+        if len(dirs) > 1 and len(dirs) != 2:
```

Comment:
The condition `len(dirs) > 1 and len(dirs) != 2` is redundant. You can simplify this by just checking `len(dirs) > 1` since the logic for `len(dirs) == 2` is handled separately.